exec/java-exec/src/main/codegen/templates/ComparisonFunctions.java

```diff
@@ -215,6 +192,36 @@ public void eval() {
     }
   }
 
+  <#-- IS_DISTINCT_FROM function -->
+  @FunctionTemplate(names = {"is_distinct_from", "is distinct from" },
```

Comment:
If adding new functions is a concern, I can make the ```RelOptUtil#splitJoinCondition``` to identify rewritten ```IS NOT DISTINCT FROM``` functions also.